src/web/components/Home/RotateAPIKeysCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Link } from 'react-router-dom';
+
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+import {
+  deduplicateStrings,
+  formatStringsWithSeparator,
+  formatUnixDate,
+} from '../../utils/textHelpers';
+
+import './RotateAPIKeysCard.scss';
+
+type RotateApiKeysCardProps = Readonly<{
+  apiKeysToRotate: ApiKeyDTO[];
+}>;
+
+function RotateApiKeysCard({ apiKeysToRotate }: RotateApiKeysCardProps) {
+  const getDatesCreated = () => {
+    const datesCreated = apiKeysToRotate.map((apiKey) => formatUnixDate(apiKey.created).toString());
+    const datesCreatedString = formatStringsWithSeparator(deduplicateStrings(datesCreated));
+    return datesCreatedString;
+  };
+
+  return (
+    <div className='rotate-api-keys-card'>
```

Comment:
I feel like this should use the same shared component as `<DocumentationCard>`. Perhaps they can both use `<Card>`?What do you think?

Reply:
Yep - changed all 3 components to use Card as their outer tag instead of <div> and <RotateApiKeysCard> and <DocumentationCard> both didn't need their own class names in that outer tag, the css for Card takes care of it.